### PR TITLE
PlayMobile Channel (Uzbekistan)

### PR DESCRIPTION
**Pull request by johncordeiro**
This PR adds the SMS Aggregator PlayMobile (http://playmobile.uz) from Uzbekistan.

### CODE REVIEW

File: temba/channels/types/playmobile/views.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        data = form.cleaned_data
+        org = user.get_org()
+
+        if not org:  # pragma: no cover
+            raise Exception(_("No org for this user, cannot claim"))
+
+        from .type import PlayMobileType
+
+        config = {
+            PlayMobileType.CONFIG_SHORTCODE: data["shortcode"],
```
Comment (norkans7) [inline temba/channels/types/playmobile/views.py]: why is the shortcode should added in the config while we saved it in the address field?

File: temba/channels/types/playmobile/type.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    )
+
+    configuration_urls = (
+        dict(
+            label=_("Receive URL"),
+            url="https://{{ channel.callback_domain }}{% url 'courier.pm' channel.uuid 'receive' %}",
+            description=_(
+                "To receive incoming messages, you need to set the receive URL for your Play Mobile account."
+            ),
+        ),
```
Comment (norkans7) [inline temba/channels/types/playmobile/type.py]: No status delivery URL?

File: temba/channels/types/playmobile/views.py (hunk 1 above)
Comment (edudouglas) [reply temba/channels/types/playmobile/views.py]: Done. Shortcode saved in the address field.

**Comment by edudouglas** [comment]
Hi @norkans7, you can review this PR, please? Thanks.

File: temba/channels/types/playmobile/type.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+from temba.channels.types.playmobile.views import ClaimView
+from temba.contacts.models import TEL_SCHEME
+
+
+class PlayMobileType(ChannelType):
+    """
+    A Play Mobile channel (http://playmobile.uz/)
+    """
+
+    CONFIG_BASEURL = "base_url"
```
Comment (norkans7) [inline temba/channels/types/playmobile/type.py]: I think you can use the constants on Channel instead of re declaring these here

**Comment by WilliamTales** [comment]
@nicpottier can you please take a look at code review as requested by @norkans7? Uzbekistan CO expects to start using this channel at the first week of December.

**Comment by nicpottier** [comment]
As @norkans7 mentioned, should use shared constants on Channel when available.

I assume you are deploying from your fork, so us merging should have no effect on any of your customers. If not, you should start doing so, as we cannot guarantee a merge schedule.

**Comment by WilliamTales** [comment]
hello again @nicpottier, changes made!
Please evaluate.

File: temba/channels/types/playmobile/type.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+            url="https://{{ channel.callback_domain }}{% url 'courier.pm' channel.uuid 'receive' %}",
+            description=_(
+                "To receive incoming messages, you need to set the receive URL for your Play Mobile account."
+            ),
+        ),
+    )
+
+    def is_available_to(self, user):
+        org = user.get_org()
+        return org.timezone and str(org.timezone) in ["Asia/Tashkent", "Asia/Samarkand"]
```
Comment (nicpottier) [inline temba/channels/types/playmobile/type.py]: I believe the right way to do this is define `available_timezones` on the class.

**Comment by nicpottier** [comment]
One last tweak!

**Comment by WilliamTales** [comment]
Hey @norkans7 constants were fixed as requested and @nicpottier already gave his bless xD
Could you please go ahead and evaluate again so merge can be done? Highly appreciated.

**Pull request merged**